tensilelite/Tensile/Source/lib/include/Tensile/ContractionProblemPredicates.hpp

```diff
@@ -845,6 +845,64 @@ namespace Tensile
                 }
             };
 
+            struct AmaxDStatusCheck
```

Comment:
Better to follow other's naming: "AmaxDCheck"

Reply:
Done